hypervisor/src/arch/emulator/mod.rs

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright © 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
```

Comment:
Is this a new file? If so I think it should be just Apache-2.0

Reply:
Fixed.